arelle/plugin/validate/ESEF/resources/config.xml

```diff
@@ -4,6 +4,26 @@
         xsi:noNamespaceSchemaLocation="../../../../config/disclosuresystems.xsd">
     <!-- see ../../../../config/disclosuresystem.xml for full comments -->
 
+    <DisclosureSystem
+        names="ESMA RTS on ESEF-2025|ESEF-2025-DRAFT|esef-2025-draft|ESEF-2025|esef-2025|ESEF|esef"
```

Comment:
Our practice in recent years has been to append draft to all disclosure system names until the conformance suite is finalized and passing to avoid confusion by users.

Also `ESEF|esef` (and `ESEF-Unconsolidated|esef-unconsolidated` below) conflicts with the 2024 disclosure system. They shouldn't be used for 2025 until it's ready to be promoted to the default.